lib/pure/parseutils.nim

```diff
@@ -393,7 +393,7 @@ proc captureBetween*(s: string, first: char, second = '\0', start = 0): string =
   result = ""
   discard s.parseUntil(result, if second == '\0': first else: second, i)
 
-proc integerOutOfRangeDefect() {.noinline.} =
+proc integerOutOfRangeError() {.noinline.} =
```

Comment:
Can you please not conflate bugfixes with refactoring?

Reply:
fine, i'll move those refactorings to a separate PR